battleship.py

```diff
@@ -24,7 +24,7 @@ def check_vic(tally):
 
 
 def print_welcome():
-    pass
+    print("Welcome To Battle Ship")
```

Comment:
Excellent. For next time let's keep adding to the welcome screen to make it appealing to the player and descriptive of the game.